src/routes/user.js

```diff
@@ -15,6 +16,12 @@ const router = Router()
 
 router.post('/', create)
 router.get('/', validateAuthentication, getAll)
+router.get(
+  '/studentWithoutCohort',
```

Comment:
This URL is a bit unusual - in REST, your URLS should usually reflect a "resource" - or a "thing" in the system. It's also unusual to see camel case in REST API urls (for example see: https://developer.spotify.com/documentation/web-api/reference/#/operations/get-recommendation-genres, usually hyphens are used). 

We could make a new controller to support requests like "/student?cohort=none" , which would be quite nice, or as an alternative, simply change the existing URL to something like "/user/student-no-cohort". Up to you.

Reply:
That's a good idea, we have changed the endpoint to a "/student?cohort=none" functionality. 
And changed the endpoint to not included camelCase.